feat: :sparkles: add REDCap field naming conventions (#222)
onlimit-study/dmp: sections/metadata.qmd

sources of data include:

- Participant input via the Liva Healthcare app and responses to questionnaires.
- Medical devices and scanners, which will produce structured data files. A
  detailed list of scanners and devices will be maintained as part of the
  technical documentation.
- Additional sensors and site-level data collection, uploaded as CSV files by
  study staff.

Data generation will occur throughout the two-year participation period for each
individual. While there are plans to link study data with register data, this
work will take place on Statistics Denmark's servers. Access to the data will be
strictly controlled through GenomeDK. Researchers must submit an application to
the Publications Panel to gain access, ensuring transparency and governance.

<!--TODO Do we want to publish the publications guidance somewhere, and if so, do we want to link it here-->

## Metadata strategy

To ensure data usability and interoperability, comprehensive metadata will be
created and maintained throughout the study:

- Field-level documentation: We will write detailed variable descriptions using
  the *Field Annotation* functionality in REDCap. This includes adherence to

>   naming conventions defined by internal rules.

Reviewer: ```suggestion
  naming conventions defined by some naming rules (see below).
```

  <!--TODO Check with Anne-Ditte if they are happy for those guidelines to be made public-->

- Supplier data dictionaries: All external data providers (e.g., Liva
  Healthcare, MyFood24) will supply data dictionaries in CSV format. These will
  be integrated into the metadata repository by the data architect.

- Device and sensor data: The site staff and the data architect will

>   collaboratively document the CSV filesoriginating from scanners and sensors to

Reviewer: ```suggestion
  collaboratively document the CSV files originating from scanners and sensors to
```

  ensure completeness and clarity.

- Metadata packaging: We will use software developed from the [Seedcase
  Project](https://seedcase-project.org) to generate and store metadata in a
  standardized data package format. These software will also structure the data
  in a standardized format and will run checks on the data to confirm that it
  corresponds to the metadata. The data package will be stored on GenomeDK,
  where the data will be version controlled and a backup kept in case of errors
  or issues. The metadata will be connected to GitHub to increase accessibility
  and discoverability as well as improve future collaboration on analyses of the
  data.

> ### Internal field naming conventions in REDCap

Reviewer: ```suggestion
### Field naming conventions in REDCap
```
